datafusion/expr/src/expr_rewriter.rs

```diff
@@ -84,6 +84,8 @@ pub enum RewriteRecursion {
 /// # See Also:
 /// * [`Expr::accept`] to drive a rewriter through an [`Expr`]
 /// * [`rewrite_expr`]: For rewriting an [`Expr`] using functions
+///
+/// [`Expr::accept`]: crate::expr_visitor::ExprVisitable::accept
```

Comment:
this isn't the right link, but i couldn't figure out how to link to the `Expr` method implementation instead of just the `ExprVisitable` trait method itself.